src/commands/convert/msft_secure2hdf.ts

```diff
@@ -0,0 +1,131 @@
+import {Command, Flags} from '@oclif/core'
+import fs from 'fs'
+import https from 'https'
+import {MsftSecureScoreResults as Mapper} from '@mitre/hdf-converters'
+import {checkSuffix} from '../../utils/global'
+import {ClientSecretCredential} from '@azure/identity'
+import {Client, ClientOptions, PageIterator, PageIteratorCallback} from '@microsoft/microsoft-graph-client'
+import {
+  SecureScore,
+  SecureScoreControlProfile,
+} from '@microsoft/microsoft-graph-types'
+import {TokenCredentialAuthenticationProvider} from '@microsoft/microsoft-graph-client/authProviders/azureTokenCredentials'
+import _ from 'lodash'
+
+function processInputs(
+  scoreDoc: SecureScore,
+  profiles: { value: SecureScoreControlProfile[] },
+  output: string,
+  withRaw: boolean,
+): any {
```

Comment:
this function has no outputs so there is no return type needed here.  if you want to put something, i think the appropriate type is 'void'

in any case, avoid using 'any' to as high a degree as possible since it more or less turns off the type system